src/main/java/org/polyfill/Services/UserAgentParserService.java

```diff
@@ -5,30 +5,39 @@
 import net.sf.uadetector.UserAgentStringParser;
 import net.sf.uadetector.VersionNumber;
 import net.sf.uadetector.service.UADetectorServiceFactory;
-import org.polyfill.components.userAgent.UserAgentImpl;
-import org.polyfill.interfaces.ConfigLoaderServiceInterface;
+import org.polyfill.components.UserAgentImpl;
+import org.polyfill.interfaces.ConfigLoaderService;
 import org.polyfill.interfaces.UserAgent;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.stereotype.Service;
 
+import javax.annotation.PostConstruct;
 import java.io.IOException;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 
-/**
- * Created by reinier.guerra on 10/12/16.
- */
 @Service
 public class UserAgentParserService {
 
-    private static final String USER_AGENT_ALIAS_PATH = "./configs/userAgentAliases.json";
     private final UserAgentStringParser uaParser = UADetectorServiceFactory.getResourceModuleParser();
-    private Map<String, Object> userAgentAliases;
+    private final String USER_AGENT_ALIASES_FILE = "./configs/userAgentAliases.json";
 
     @Autowired
-    public UserAgentParserService(JSONConfigLoaderService jsonConfigLoaderService) {
-        this.userAgentAliases = getConfig(jsonConfigLoaderService, USER_AGENT_ALIAS_PATH);
+    @Qualifier("json")
+    private ConfigLoaderService configLoaderService;
+
+    private Map<String, Object> userAgentAliases;
+
+    @PostConstruct
```

Comment:
PostConstruct allows us to initialize service after other services/beans are initialized properly. It'll also make tests easier to write.